src/functionComp/InputTodo.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react"
+
+const InputTodo = (props) => {
+  const [inputText, setInputText] = useState({
+    title: "",
+  })
+  const onChange = e => {
+    setInputText({
+      ...inputText,
+      [e.target.name]: e.target.value,
+    })
+  }
+  const handleSubmit = e => {
+    e.preventDefault();
+    if (inputText.title.trim()) {
+        props.addTodoProps(inputText.title)
+        setInputText({
+          title: "",
+        }) 
+      } else {
+        alert("Please write item")
```

Comment:
using alerts is against javascript best practices, kindly use  another element, for instance, a modal